apps/docs/src/examples/renderers/CSS2DRenderer/Scene.svelte

```diff
@@ -0,0 +1,59 @@
+<script lang="ts">
+  import { tick } from 'svelte'
+  import { T, useFrame, useRender, useThrelte } from '@threlte/core'
+  import { OrbitControls } from '@threlte/extras'
+  import { CSS2DRenderer } from 'three/addons/renderers/CSS2DRenderer.js'
+  import CssObject from './CssObject.svelte'
+
+  // Set up the CSS2DRenderer to run in a div placed atop the <Canvas>
+  const element = document.querySelector('#css-renderer-target')
+  const cssRenderer = new CSS2DRenderer({ element })
+
+  const { size } = useThrelte()
```

Comment:
Consider moving this before line 9 so that all the `cssRenderer` lines are together.